server/src/completions/completion-item-data.d.ts

```diff
@@ -0,0 +1,9 @@
+/* --------------------------------------------------------------------------------------------
```

Comment:
I don't see license suggestions when the assigment is empty.
```
# Press Ctrl+Space in the quotes
LICENSE = ""
# Although this one works
LICENSE:append = " MIT BS"
```
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/aafde80b-ebcf-4157-b472-5972db214c47)

Reply:
Otherwise excelent for showing deprecation and documentation 
![image](https://github.com/yoctoproject/vscode-bitbake/assets/19970889/7b7f0a47-f1d8-4a64-8f2b-0638d2e98423)